onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -258,6 +248,69 @@ TEST_P(ModelTest, Run) {
 #endif
       {"mask_rcnn_keras", "this model currently has an invalid contrib op version set to 10", {}}};
 
+  std::set<BrokenTest> bfloat16_string_tests = {
```

Comment:
nit: not all tests in this list are related to bfloat16, there may be a better name

Reply:
removed and moved to their original place